tests/utils/test_generate.py

```diff
@@ -50,7 +50,7 @@ def test_batched():
     ids = tokenizer.convert_tokens_to_ids(tokenizer.tokenize("ABCDE"))
     first = torch.tensor(ids)
     second = torch.tensor(ids)
-    ids = torch.stack((first, second), dim=0)
+    ids = [first, second]
```

Comment:
are we removing the ability to call generate on batched rows? we have inference scripts that rely on that ability

Reply:
Would we just assume then that the prompts are all of the same length?

Reply:
I mean, that's what all the existing scripts do, right? with this change, I think all the existing scripts will fail/error until/unless updated to conform to this API change

Reply:
Ah yes, I think we'd need to update the other script (not just inference then) as well. Would you prefer we just assume tensor as batch all have same length per prompt